.dev-lib

```diff
@@ -1,5 +1,2 @@
-# Variables needed to generate readme
-
-README_MD_TITLE="AMP Plugin for WordPress"
-ASSETS_DIR=wp-assets
+# This is used by wp-dev-lib/scripts/check-diff.sh but the functionality is no longer used, so it could be removed.
```

Comment:
This file could now be removed, along with the `xwp/wp-dev-lib` dependency since it's not being used anywhere else.

Reply:
Removed in 2f999e34f

Reply:
This will require updating the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) wiki page as well.

Reply:
Done: https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines/_compare/b16b2699c8f54d5adf489d4ad8bf69267606e04b...6ee5670c1a22f08c816114f36060015605d76e95